scripts/install_third_party_libs.py

```diff
@@ -155,9 +159,13 @@ def compile_protobuf_files(proto_files_paths):
     buf_path = os.path.join(
         BUF_DIR,
         BUF_DARWIN_FILES[0] if common.is_mac_os() else BUF_LINUX_FILES[0])
-    for path in proto_files_paths:
-        command = [
-            buf_path, 'generate', path]
+    for is_frontend_required, path in proto_files_paths:
```

Comment:
Not clear to me what `is_frontend_required` means. Try a different variable name?

Taken literally, this seems to mean "is the entire frontend required" which doesn't make sense to me in context.

Also, for booleans, try not to start their varnames with a verb.

Reply:
Updated with the `generate_js_and_ts_files`, starting with a noun.